src/utils/session-id.js

```diff
@@ -2,6 +2,7 @@ import { v4 as uuidv4 } from 'uuid'
 import Cookie from 'js-cookie'
 
 const COOKIE_EXPIRY_DAYS = 7
+const COOKIE_PREFIX = 'openverse_'
```

Comment:
This is a small naming improvement but also meant to clear any cached cookies with the old cookie policies.